code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/AepUiState.kt

```diff
@@ -0,0 +1,6 @@
+package com.adobe.marketing.mobile.aepcomposeui.aepui.state
+
+sealed interface AepUiState
+
+data class SmallImageUIState(val title: String?, val dismissed: Boolean = false) : AepUiState
```

Comment:
Let's move this to its own file
It will need to be changed in future PRs to reflect actual values the state needs to hold

Reply:
Never mind just realized this is a sealed interface